otter/convergence/gathering.py

```diff
@@ -205,23 +205,28 @@ def _lb_configs_from_metadata(server):
 
     # throw away any value that is malformed
 
-    def parse_config(config, key):
-        if config.get('type') != 'RackConnectV3':
-            lbid = k[len(_key_prefix):]
+    def parse_config(lb_type, lb_id, configs):
```

Comment:
Yes, it's because they're thrown out.  I was going to either create CLBDescription or RCv3Description here, but we don't actually have the RCv3Descriptions or nodes yet as models, so nothing can be done with the RCv3 data yet.